test/e2e/network/dns_common.go

```diff
@@ -235,6 +242,21 @@ func (t *dnsTestCommon) deleteUtilPod() {
 	}
 }
 
+// deleteCoreDNSPods manually deletes the CoreDNS pods to apply the changes to the ConfigMap.
```

Comment:
Yes, reload plugin would work. However it is slow : it can take up to 2mn20 to have the reload effective.

Reply:
Ack, we can live with this for now.

> it can take up to 2mn30 to have the reload effective

Curious why 2m30, would this be changed to a pull model in the future?

Reply:
Oops I meant push..

Reply:
@MrHohn : it is not expected to be changed. 
The current behavior to update the config of CoreDNS is to associate a persitent file named "Corefile" with the configmap "coredns". 
The delay to have the persistent file updated after edition of the Configmap can go up to 2mn.
The delay to "reload" the configuration file in CoreDNS is tuned to 30sec (but could be decrease down to 2 sec).

Overall the most important delay is coming from Kubernetes' mechanism to propagate the update to the persistent file.